### ISSUE TITLE
& doesn't work properly when pseudo class wrapped by @at-root.

### ISSUE THREAD

**Issue by kryvonos-v** [sass/node-sass#2158]
I've got the following code:
```scss
.tooltip {
  &__arrow {
    @at-root .tooltip--arrow-down .tooltip__arrow {
      &:before {
        @debug #{&};
      }
    }
  }
}
```
Expected that the code about will output to console:
`.tooltip--arrow-down .tooltip__arrow:before`
But I get:
`.tooltip--arrow-down .tooltip__arrow .tooltip--arrow-down .tooltip__arrow:before`.

I've tried to compile that code with Ruby Sass and everything works as expected. I guess that the bug comes from LibSass, but I don't really sure about it.

- node-sass 4.7.1

**Comment by nschonni** [comment]
Yes, as mentioned in the issue template, compilation issues should be filed upstream. If you open an issue there you can link it back here.

**Comment by xzyfer** [comment]
Fixed in 4.8.0
